README.rst

```diff
@@ -70,6 +70,8 @@ This client supports the following Google Cloud Platform services at an Alpha_ q
 If you need support for other Google APIs, check out the
 `Google APIs Python Client library`_.
 
+.. _Alpha: https://cloud.google.com/terms/launch-stages
+.. _Beta: https://cloud.google.com/terms/launch-stages
```

Comment:
I could remove the reference from

>This client supports the following Google Cloud Platform services at a Beta_ quality level:

then add something like this at the bottom of the main section.

>If you would like to learn more about the launch stages for the
Google Cloud Platform please visit https://cloud.google.com/terms/launch-stages.